src/client/dfuse/dfuse.h

```diff
@@ -138,6 +138,75 @@ struct dfuse_obj_hdl {
 	bool                      doh_kreaddir_finished;
 };
 
+/* Readdir support.
```

Comment:
Should this be documented in the docs somewhere instead of buried in a `.h`? (that most people won't read)

Reply:
It's only of interest to people working on the code I think.  There's no public API here.